.travis.yml

```diff
@@ -32,9 +82,9 @@ before_script:
       composer global require "phpunit/phpunit=4.8.*"
     fi
   - |
-    composer global require wp-coding-standards/wpcs
```

Comment:
sure we dont need to require `wp-coding-standards/wpcs`?

Reply:
Yes, it's a dependency of VIPWPCS.